tests/canopen/test_canopen_network.py

```diff
@@ -11,6 +35,33 @@ def test_connect_to_slave(connect_to_slave):
     assert fw_version is not None and fw_version != ''
 
 
+@pytest.mark.canopen
+def test_connect_to_slave_target_not_in_nodes(read_config):
+    protocol_contents = read_config['canopen']
```

Comment:
Maybe we should create a fixture that provides us with the configuration content of the corresponding protocol. In case the protocol name is changed.